src/include/wiredtiger.in

```diff
@@ -5891,13 +5891,16 @@ extern int wiredtiger_extension_terminate(WT_CONNECTION *connection);
  * store tombstone
  */
 #define	WT_STAT_CONN_CURSOR_NEXT_HS_TOMBSTONE		1252
+/*!
+ * cursor: cursor next calls that skip greater than 1 and fewer than 100
+ * entries
+ */
+#define	WT_STAT_CONN_CURSOR_NEXT_SKIP_LT_100		1253
```

Comment:
It is weird that it reordered the stats even thought the statistic name didn't change. This is an annoying question but do you know why it did that?

Reply:
Good question. I did some digging and found out that the `s_all` script runs `stat.py`, which orders the statistics by description instead of by name: `# The list must be sorted by description. ` I've reordered the way it appears in `stat_data.py `for consistency with the ordering across the other files: [31bba91](https://github.com/wiredtiger/wiredtiger/pull/8419/commits/31bba9187fa430a898a6fa3fb0cd67152d12405e)

Reply:
Weird, but okay.